server/services/generateReportService.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+//@ts-ignore
+import { Legacy } from 'kibana';
+import { Server, Request, ResponseToolkit } from 'hapi';
+import { RequestParams } from '@elastic/elasticsearch';
+import puppeteer from 'puppeteer';
+import imagesToPdf from 'images-to-pdf';
+import fs from 'fs';
+import { v1 as uuidv1 } from 'uuid';
+
+// TODO: change the following accordingly
+// import { AcknowledgedResponse, AddPolicyResponse, AddResponse, CatIndex, GetIndicesResponse, SearchResponse } from "../models/interfaces";
```

Comment:
Can we remove these comments in this file?

Reply:
will do